### ISSUE TITLE
Registered assets are not properly displayed

### ISSUE THREAD

**Issue by eckelj** [Blockstream/green_android#198]
I send a transaction of tokens of a registered asset from my wallet to another (Green Android) wallet.

The transaction looks like this from where I sent it from:

```
elements-cli gettransaction cb86fd8b1154590e1bf9b9f1a124b6c921c44727c1d49a54712103215bd332fb
{
  "amount": {
    "RDDL": -46875.00000000,
    "bitcoin": 0.00000000
  },
  "fee": {
    "bitcoin": -0.00000382
  },
  "confirmations": 997,
  "blockhash": "26346dc2a3667fe5cfb252e0dc63bc16ef92a3effe4cccd7324b4b6aa6b5edd3",
  "blockheight": 2758325,
  "blockindex": 1,
  "blocktime": 1709824058,
  "txid": "cb86fd8b1154590e1bf9b9f1a124b6c921c44727c1d49a54712103215bd332fb",
  "walletconflicts": [
  ],
  "time": 1709824120,
  "timereceived": 1709824120,
  "bip125-replaceable": "no",
  "details": [
    {
      "address": "ex1qa90tavgd074zsx26yvxkdermhrj660yrk9w8qa",
      "category": "send",
      "amount": -46875.00000000,
      "amountblinder": "---------------------------------------------------------------------------------------------------",
      "asset": "72a504dbd846e604c03adfbbc4f596922a65a7496128d8f3dc9bf7132a736501",
      "assetblinder": "---------------------------------------------------------------------------------------------------",
      "vout": 0,
      "fee": 0.00000382,
      "abandoned": false
    }
  ],
  "hex": "0200
....
```
We registered the asset beforehand so that RDDL maps to asset ID 72a504dbd846e604c03adfbbc4f596922a65a7496128d8f3dc9bf7132a736501.

On Green Wallet, it looks as follows 
![image](https://github.com/Blockstream/green_android/assets/1703752/d7ef1593-8f1b-40f6-93ca-631a4433f115)

The amount appears to be written in 4.687.500.000.000 (SAT) instead of 46875.00000000 (with a precision of 8), and the asset is not resolved. Instead, you can see the first characters of 72a504dbd846e604c03adfbbc4f596922a65a7496128d8f3dc9bf7132a736501. 

The recipient is confused. How can we solve this issue? The asset-registration service isn't responsive. Would it be possible to register assets explicitly on Green Wallet to avoid or work around these corner cases?

**Comment by angelix** [comment]
Please use the newest version 4.0.25

An asset cache issue was fixed in the latest version.

**Comment by angelix** [comment]
Version 4.0.25 is already available.

**Comment by angelix** [comment]
Fixed in f01fe1d500d7bad3c1b2adadd16942e5f4cbd05d
